test/Billing.Test/Billing.Test.csproj

```diff
@@ -1,4 +1,5 @@
-<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <Sdk Name="Bitwarden.Server.Sdk" />
```

Comment:
Was this required? I'm both a little surprised you referenced it in a test project and surprised you changed it to target the web SDK.

Reply:
This is a really old PR but I don't remember doing this, and the existing test code couldn't have ran without `.Web` since there are some status code references -- not sure how this is here.

L2 isn't necessary so I am building up changes at #196 that I can eventually merge.